apps/ledger-live-mobile/src/screens/Swap/Form/Modal/Confirmation.tsx

```diff
@@ -109,6 +109,9 @@ export function Confirmation({
           provider,
           swapId,
           transactionId: operation.hash,
+          sourceCurrencyId: swapTx.current.swap.from.account?.id,
+          targetCurrencyId: swapTx.current.swap.to.account?.id,
+          hardwareWalletType: deviceMeta.device.modelId,
```

Comment:
please add swapType as well. We want to have success vs failure rate on all these dimensions

Above (postSwapCancelled({ provider, swapId });), we should have the same data as in other cancel. You can use a new specific status code (DISABLE_TRANSACTION_BROADCAST=-1 for instance, if not already used, plus message = transaction broadcast disabled, step = the broadcast step, etc.)